src/Phinx/Db/Adapter/AbstractAdapter.php

```diff
@@ -258,4 +263,56 @@ public function isDryRunEnabled()
 
         return ($input && $input->hasOption('dry-run')) ? (bool)$input->getOption('dry-run') : false;
     }
+
+    /**
+     * Adds user-created tables (e.g. not phinxlog) to a cached list
+     *
+     * @param string $tableName The name of the table
+     * @return void
+     */
+    public function addCreatedTable($tableName)
+    {
+        if (substr_compare($tableName, 'phinxlog', -strlen('phinxlog')) !== 0) {
+            $this->createdTables[] = $tableName;
+        }
+    }
+
+    /**
+     * Updates the name of the cached table
+     *
+     * @param string $tableName Original name of the table
+     * @param string $newTableName New name of the table
+     * @return void
+     */
+    public function updateCreatedTableName($tableName, $newTableName)
+    {
+        $key = array_search($tableName, $this->createdTables);
+        if ($key !== false) {
+            $this->createdTables[$key] = $newTableName;
+        }
+    }
+
+    /**
+     * Removes table from the cached created list
+     *
+     * @param string $tableName The name of the table
+     * @return void
+     */
+    public function removeCreatedTable($tableName)
+    {
+        if (($key = array_search($tableName, $this->createdTables)) !== false) {
```

Comment:
I was also referring to all other cases of this :)

Reply:
Agreed, sorry about missing this when I quickly glanced through the other changes.